docs/src/modules/java/examples/testkit/java-eventsourced-shopping-cart/src/main/java/com/example/shoppingcart/ShoppingCart.java

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
why adding those files here? you can use the existing https://github.com/lightbend/akkaserverless-java-sdk/tree/sdk-codegen-dev/samples/java-eventsourced-shopping-cart

when you want to view the docs locally you can use
```
make examples dev-html
```

Reply:
yeah, I totally forgot. Too many holidays... :) I just had a conversation with @johanandren and told me the same. I'll add some tags to the original files